### PR TITLE
adds support for letters

### PR DESCRIPTION
**Pull request by mgartner**
@lob/api-engineering-team

### CODE REVIEW

**Comment by leore** [comment]
coverage decreased. Ping when passing

**Comment by mgartner** [comment]
Something is wrong with coveralls because those lines have never been covered as far as I know.

**Comment by pon** [comment]
LGTM

File: spec/lob/v1/letter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  describe "create" do
+    it "should create a letter with address_id" do
+      VCR.use_cassette('create_letter_with_address_id') do
+        new_address = subject.addresses.create @sample_address_params
+
+        new_letter = subject.letters.create(
+          description: "TestLetter",
+          color: true,
+          file: "https://s3-us-west-2.amazonaws.com/lob-assets/100_101_doc.pdf",
```
Comment (leore) [inline spec/lob/v1/letter_spec.rb]: change this to correct link.

File: spec/lob/v1/letter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+  describe "find" do
+    it "should find a letter" do
+      VCR.use_cassette('find_letter') do
+        new_address = subject.addresses.create @sample_address_params
+
+        new_letter = subject.letters.create(
+          description: "TestLetter",
+          color: true,
+          file: "https://s3-us-west-2.amazonaws.com/lob-assets/100_101_doc.pdf",
```
Comment (leore) [inline spec/lob/v1/letter_spec.rb]: change link

**Pull request merged**